### ISSUE TITLE
crypto.randomUUID() not recognized correctly

### ISSUE THREAD

**Issue by aczekajski** [amilajack/eslint-plugin-compat#649]
According to MDN, Safari did not support `randomUUID` until 15.4 (https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID).

My eslint config:
```
{
    "plugins": [
      "compat"
    ],
    "extends": [
      "plugin:compat/recommended"
    ]
  }
```

browserlist:
```
"browserslist": {
    "production": [
      "safari 14"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
```

Code:
```
crypto.randomUUID();
```

No warning is raised on that code while it in fact does throw an error in Safari 14. Other things unsupported in Safari 14 (for example `structuredClone`) are properly raised so the config is not wrong.

**Comment by vsalomaki** [comment]
I stumbled on the exact same issue, added PR for it. It appeared that it was comparing "crypto" to "Crypto"
